coss/discourse/models.py

```diff
@@ -0,0 +1,15 @@
+from django.db import models
+
+from wagtail.wagtailadmin.edit_handlers import FieldPanel
+from wagtail.wagtailsnippets.models import register_snippet
+
+@register_snippet
+class DiscourseCategoryLatest(models.Model):
+    category_id = models.PositiveSmallIntegerField()
```

Comment:
This'll be a snippet which fetches the latest topics from a defined category. Maybe I should have included the logic to fetch data from the Discourse API in this commit to make it sensible?